awx/api/serializers.py

```diff
@@ -5,6 +5,7 @@
 import copy
 import json
 import logging
+from multiprocessing.sharedctypes import Value
```

Comment:
Doesn't look like this is used anymore.

Reply:
Thank you so much!! I will get this fixed as soon as I fix my rebase!